### DISCUSSION TITLE
error when execute "docker compose up -d"

### DISCUSSION THREAD

**Discussion by kronicklez** [akvorado/akvorado#1479] [Q&A]
Hi @vincentbernat /expert,

Is there any changes on version 1.11.2 coz now i'm cannot enable docker. 

root@ubuntu20:~/akvorado# docker compose up -d
**Top-level object must be a mapping**

Appreciate your feedback

**Comment by vincentbernat** [comment]
Can you try to comment the line about `docker-compose-local.yml` in `.env`?
